CanDIG/candigv2-ingest
# Ingest into katsu without auth enabled

Added flags to katsu scripts to allow a person to run without auth enabled.

To test:
* in the CanDIGv2/.env file, you’ll need to update line 207 to say CANDIG_AUTHORIZATION=""
* go to CanDIGv2/lib/chord-metadata/chord_metadata_service/ and git pull, then git checkout daisieh/no-auth
* cd back to the CanDIGv2 main directory and run make build-chord-metadata and make compose-chord-metadata

Now you should be able to run the katsu scripts with the `--no_auth --katsu_url http://docker.localhost:8008` flags, like so:
```
python katsu_ingest.py  --input /test.json --dataset test5 --no_auth --katsu_url http://docker.localhost:8008
```
